test/controllers/api/v1/exchange_rates_controller_test.rb

```diff
@@ -0,0 +1,166 @@
+require "test_helper"
+
+module Api
+  module V1
+    class ExchangeRatesControllerTest < ActionDispatch::IntegrationTest
+      setup do
+        # Crear conjunto de datos de prueba
+        @from = "USD"
+        @to = "EUR"
+        
+        # Fechas para pruebas
+        @today = Date.current
+        @yesterday = 1.day.ago.to_date
+        @last_week = 7.days.ago.to_date
+        
+        # Crear registros de prueba
```

Comment:
All those comments are pretty useless for me, as the code is pretty self explaining. But maybe they can help the IA, so not complaining about them